src/python/bot/tasks/fuzz_task.py

```diff
@@ -1301,6 +1301,14 @@ def run_engine_fuzzer(engine_impl, target_name, sync_corpus_directory,
   options = engine_impl.prepare(sync_corpus_directory, target_path, build_dir)
 
   fuzz_test_timeout = environment.get_value('FUZZ_TEST_TIMEOUT')
+  additional_processing_time = engine_impl.fuzz_additional_processing_timeout(
+      options)
+  fuzz_test_timeout -= additional_processing_time
```

Comment:
Why do we do -(-X), why not just return positive val from this function and add it here.

Reply:
This is a positive value. We negate it in the implementation of libFuzzer's fuzz_additional_processing_timeout because libfuzzer.get_fuzz_timeout() returned a negative value. 

We can clean up libfuzzer.get_fuzz_timeout to return a positive value in the first place, but that needs to wait for further refactoring (including AFL->engien refactor) so that we can get rid of some env var overrides used for testing  (e.g. HARD_TIMEOUT_OVERRIDE).

Reply:
ok sounds good.